src/index.ts

```diff
@@ -104,12 +114,126 @@ class ComputerSpeakers implements Accessory {
           this.getVolume.bind(this, logarithmic)
         )
     }
+
+    if (!!delta) {
```

Comment:
Should both work the same way.  If `delta` is `undefined` or `0` or `null`, then `!delta` is `true`, since `undefined`, `0`, and `null` are each equal to `false` w.r.t. simple comparison `==`.